### PR TITLE
[BHE - 2019] date and venue

### PR DESCRIPTION
**Pull request by devopsdaysbh**
Now finally with dates and stuff

### CODE REVIEW

File: data/events/2019-belo-horizonte.yml [hunk 1]
```diff
@@ -10,29 +10,27 @@ ga_tracking_id: "UA-124055941-1" # If you have your own Google Analytics trackin
 #   variable: 2019-01-05T23:59:59+02:00
 # Note: we allow YYYY-MM-DD for backward compatibility, but it can lead to unexpected behaviors (like your event disappearing from the front page during your last day)
 
-startdate: # The start date of your event. Leave blank if you don't have a venue reserved yet.
-enddate:  # The end date of your event. Leave blank if you don't have a venue reserved yet.
+# All dates are in unquoted 2018-MM-DD, like this:   variable: 2016-01-05
+startdate: 2018-09-28 # The start date of your event. Leave blank if you don't have a venue reserved yet.
+enddate: 2018-09-28 # The end date of your event. Leave blank if you don't have a venue reserved yet.
```
Comment (bridgetkromhout) [inline data/events/2019-belo-horizonte.yml]: We recommend the full datetime - see comments in the file.

File: data/events/2019-belo-horizonte.yml [hunk 2]
```diff
@@ -100,8 +98,8 @@ team_members: # Name is the only required field for team members.
 #    # facebook: ""
 #    # linkedin: ""
 #    # website: ""
-organizer_email: "organizers-belo-horizonte-2018@devopsdays.org" # Put your organizer email address here
-proposal_email: "proposals-belo-horizonte-2018@devopsdays.org" # Put your proposal email address here
+organizer_email: "organizers-belo-horizonte-2019@devopsdays.org" # Put your organizer email address here
+proposal_email: "proposals-belo-horizonte-2019@devopsdays.org" # Put your proposal email address here
```
Comment (bridgetkromhout) [inline data/events/2019-belo-horizonte.yml]: Whomever merges, please make certain these are set up.

Comment (nathenharvey) [reply data/events/2019-belo-horizonte.yml]: Confirmed that organizers-belo-horizonte-2019@devopsdays.org and proposals-belo-horizonte-2019@devopsdays.org are both set-up.

File: data/events/2019-belo-horizonte.yml (hunk 1 above)
Comment (bridgetkromhout) [reply data/events/2019-belo-horizonte.yml]: I fixed the datetime string (which also said 2018).

Comment (devopsdaysbh) [reply data/events/2019-belo-horizonte.yml]: Thanks a lot @bridgetkromhout for the help. For some reason I left this behind, idk why. Everything than that is right, fortunately

**Pull request merged**